src/common.ts

```diff
@@ -243,16 +243,34 @@ export function isSemanticVersionGreaterThan(appVersion_a: string, appVersion_b:
 	return false;
 }
 
+/**
+ * Returns true if the date is after the day_start and before the day_end
+ * @param day_start the day when the range starts
+ * @param day_end the day when the range ends (not included)
+ * @param date the date to check
+ */
+export function isDateInRangeOfDays(day_start: Date, day_end: Date, date: Date): boolean {
```

Comment:
Can we have tests for `isDateInRangeOfDays`?

Reply:
There we have
ec6596fcc8b4c532efd956aed5b1941f7b1ecb12